concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -141,14 +141,14 @@ static PACKET_TYPE_FROM_INT: &[PacketType] = &[
 ];
 
 impl TryFrom<u8> for PacketType {
-    type Error = failure::Error;
+    type Error = anyhow::Error;
 
     #[inline]
-    fn try_from(value: u8) -> Fallible<PacketType> {
+    fn try_from(value: u8) -> anyhow::Result<PacketType> {
         PACKET_TYPE_FROM_INT
             .get(value as usize)
             .copied()
-            .ok_or_else(|| format_err!("Unsupported packet type ({})", value))
+            .ok_or_else(|| anyhow!("Unsupported packet type ({})", value))
```

Comment:
With `anyhow` this can be simplified to `.get(value as usize).copied().context(format!("Unsupported packed type ({})", value))

Not sure if that's better or not.

Reply:
We should probably do it anyway for consistency (wrt. the other comments you have).